### ISSUE TITLE
Removing Mock server details from API base URL

### ISSUE THREAD

**Issue by smitabh84** [stoplightio/elements#2042]
## Describe the bug
When I am adding the server details, the mock server details are being added by default. Can you help me with a way to remove these mock server details altogether from my public APIs.

## To Reproduce
visit https://www.nutanix.dev/api_references/era/#/YXBpOjE2MjkwNjE5-api-reference

![image](https://user-images.githubusercontent.com/66934062/155288827-aa8da5f8-e8f4-4eb4-a6bd-7dbfc2ee7b45.png)
## Additional context

**Screenshots**

**Environment (remove any that are not applicable):**
 - Library version: [e.g. 3.0.0]
 - OS: [e.g. Windows 7]
 - Browser: [e.g. Chrome 61]
 - Platform: [desktop or web]

**Comment by mnaumanali94** [comment]
Removing mock server is a feature enhancement. An issue here is fine but can you also please add your insights here: https://roadmap.stoplight.io/c/160-show-hide-mocking-in-try-it

The default is the live server though.

**Comment by smitabh84** [comment]
Hi Nauman,

Currently our customers are seeing both Live servers and Mock servers. My requirement is to remove mock server details from the final published documents and keep it for internal testing purpose ONLY. So, basically we need to default the server as Live server and not mock server.

The details mentioned in the https://roadmap.stoplight.io/c/160-show-hide-mocking-in-try-it [roadmap.stoplight.io]<https://urldefense.proofpoint.com/v2/url?u=https-3A__roadmap.stoplight.io_c_160-2Dshow-2Dhide-2Dmocking-2Din-2Dtry-2Dit&d=DwMCaQ&c=s883GpUCOChKOHiocYtGcg&r=UbNX3gjLAGfqIeW034NRCo6TFSpLuMGs8wS2LOBPGcg&m=wIG1wHVnQNmSvu6d3Pe2qipPr9Ii25bI1HqHiOwDq7G-tt2raQLYevdLLi04JZev&s=ZkZSA32GWRBAeuLKBs4QJaVejomWzMQW_TpgMlhU_mg&e=> seems to be different. I am not able to edit them.

Please edit them then raise for an enhancement request. Also, please share the timelines for this request to be fixed. We need to make our customers aware of the same.

Thanks.

**Comment by mnaumanali94** [comment]
@allegra-rosenbaum My bad! Considering you are using elements you should be able to hide the `hideMocking` option. https://meta.stoplight.io/docs/elements/ZG9jOjMyNjU4OTU3#configuration to hide any mention of the mock server.

**Comment by smitabh84** [comment]
Thanks Nauman,

I tried setting this up, but then the Live server details also vanishes once we set the “hidemocking” true parameter in our html code.

So we have a project where we have set the hidemocking true. Here is the link of the API page– https://www.nutanix.dev/api_references/karbon/#/b3A6NDM0MDU2MjE-update-the-certificates-used-by-the-various-components-of-the-k8s-cluster, both the mockserver and live server vanishes from the send API request section(see red arrow below).

***@***.***

Once I remove the hidemocking: true value, both the live server and mock server shows up.
Am I setting something wrong?

Thanks,
Smita,
